app/styles/component_tree.scss

```diff
@@ -123,17 +101,7 @@
 
 /**
  * Modifier
- * highlighted - children of selected component
- */
-
-.component-tree-item.component-tree-item--highlighted {
-  background-color: var(--component-highlighted-bg);
-  border-radius: 0;
-}
```

Comment:
This was originally used to subtly highlight children of a selected component. At some point that changed and was being used for the hover state. There was a visible lag.